client/src/components/Cover/PendingCoverage/PendingCoverage.js

```diff
@@ -34,6 +33,48 @@ function processData(database) {
   return listItems;
 }
 
+class PendingCov extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [
```

Comment:
We shouldn't have any hardcoding on the frontend anymore. This should be pulled from the backend like RequestHistory.